CELL TD/Assets/Scripts/UI/QuitGame.cs

```diff
@@ -0,0 +1,11 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class QuitGame : MonoBehaviour
+{
+    public void ExitGame()
+    {
+        Application.Quit();
```

Comment:
This should take you back to the level selector instead of quitting the game, please update the branch to have access to the scenes.